packages/palette/src/elements/Input/Input.tsx

```diff
@@ -55,7 +56,11 @@ interface StyledInputProps extends React.HTMLProps<HTMLInputElement> {
 /**
  * func to compute border color
  */
-export const computeBorderColor = ({ disabled, error, pseudo = null }) => {
+export const computeBorderColor = (
+  disabled: boolean,
+  error: boolean,
+  pseudo?: string
+): Color => {
```

Comment:
```js 
export const computeBorderColor: {
  disabled: boolean,
  error: boolean,
  pseudo?: string
} = ({ 
  disabled, 
  error,
  pseudo
}) => {
  ... 
}
```
this also follows closely with the react props model, which these days is the dominant pattern for working with functions.

Reply:
I opted for the interface, good feedback!!! 🤗